auth-web/src/util/common-util.ts

```diff
@@ -268,4 +269,9 @@ export default class CommonUtils {
   static trimTrailingSlashURL (url) {
     return (url) ? url.trim().replace(/\/+$/, '') : ''
   }
+
+  /** use NR request type code to get NR type from enum */
+  static mapRequestTypeCdToNrType (requestTypeCd: NrRequestTypeCodes): string {
```

Comment:
Technically you're returning `NrRequestTypes` (not string), but I prefer to consider it a string since you're displaying it. To be 100% correct you should cast (`as string`) what you're returning below.